modules/gltf/src/lib/gltf-scenegraph.d.ts

```diff
@@ -90,39 +90,53 @@ export default class GLTFScenegraph {
 
   setObjectExtension(object: object, extensionName: string, data: object): void;
 
+  // Adds a scene to the json part
```

Comment:
Please use the `/** */` syntax instead of `//`. 

I know you told me both work but I prefer to keep it consistent.

Reply:
Done in https://github.com/visgl/loaders.gl/pull/970